docker-compose.yaml

```diff
@@ -3,32 +3,32 @@
 version: "3"
 services:
   shiori:
-    build:
-      context: .
-      dockerfile: Dockerfile.compose
+    image: ghcr.io/go-shiori/shiori
```

Comment:
Please revert this to the original state, since this docker compose is meant for development purposes.

Reply:
I understand